tsconfig.json

```diff
@@ -26,8 +25,8 @@
     "importsNotUsedAsValues": "error",
     "skipLibCheck": true,
     "paths": {
+      "@public/*": ["./public/*"],
```

Comment:
Also here the `@test` folder was not that used by the codebase but had to call plenty the `public` folder